salt/_modules/metalk8s_kubeconfig.py

```diff
@@ -16,7 +16,7 @@ def __virtual__():
 
 def validate(filename,
              expected_ca_data,
```

Comment:
Not related to this PR but to me, this one should be optional as well so that a user can easily check a kubeconfig by running `salt-call metalk8s_kubeconfig.validate <my-kubeconf>`